src/sentry/api/endpoints/useravatar.py

```diff
@@ -1,81 +1,59 @@
 from __future__ import absolute_import
 
-import base64
-
-from django.conf import settings
-from PIL import Image
 from rest_framework import status
+from rest_framework import serializers
 from rest_framework.response import Response
-from six import BytesIO
-from uuid import uuid4
 
 from sentry.api.bases.user import UserEndpoint
+from sentry.api.fields import AvatarField
 from sentry.api.serializers import serialize
-from sentry.models import UserAvatar, File
-
-
-MIN_DIMENSION = 256
-
-MAX_DIMENSION = 1024
+from sentry.models import UserAvatar
+
+
+class UserAvatarSerializer(serializers.Serializer):
+    avatar_photo = AvatarField(required=False)
```

Comment:
ill plan to refactor this to conform to API naming and put it in the UserDetails endpoint in a followup patch